src/utils/mapUtils.js

```diff
@@ -102,3 +108,13 @@ export function convertTrailFeaturesToDonuts(trail) {
   }
   return trail;
 }
+
+
+export function hexToRgb(hex) {
```

Comment:
Not a mapUtil, throw it in a separate `utils` file we can use for general things.

Reply:
Done.